From 90feb101da6c2f23f7d8cacb99cd55fb16e9a39a Mon Sep 17 00:00:00 2001
From: Yumi Yukimura <me.cafebabe@gmail.com>
Date: Fri, 25 Oct 2024 03:38:25 +0800
Subject: [PATCH 1/4] init.lineage-system_ext.rc: Rename `bugreport` service

* Cuttlefish has defined one with the same name.

[ 99% 157176/157198] build out/target/product/vsoc_x86_64/host_init_verifier_outp
FAILED: out/target/product/vsoc_x86_64/host_init_verifier_output.txt
/bin/bash -c "out/host/linux-x86/bin/host_init_verifier                 -p out/target/product/vsoc_x86_64/obj/ETC/passwd_system_intermediates/passwd_system        -p out/target/product/vsoc_x86_64/obj/ETC/passwd_system_ext_intermediates/passwd_system_ext                -p out/target/product/vsoc_x86_64/obj/ETC/passwd_vendor_intermediates/passwd_vendor               -p out/target/product/vsoc_x86_64/obj/ETC/passwd_odm_intermediates/passwd_odm             -p out/target/product/vsoc_x86_64/obj/ETC/passwd_product_intermediates/passwd_product             --property-contexts=out/target/product/vsoc_x86_64/obj/ETC/plat_property_contexts_intermediates/plat_property_contexts              --property-contexts=out/target/product/vsoc_x86_64/obj/ETC/system_ext_property_contexts_intermediates/system_ext_property_contexts --property-contexts=out/target/product/vsoc_x86_64/obj/ETC/product_property_contexts_intermediates/product_property_contexts               --property-contexts=out/target/product/vsoc_x86_64/obj/ETC/vendor_property_contexts_intermediates/vendor_property_contexts          --property-contexts=out/target/product/vsoc_x86_64/obj/ETC/odm_property_contexts_intermediates/odm_property_contexts              --out_system out/target/product/vsoc_x86_64/system                --out_system_ext out/target/product/vsoc_x86_64/system_ext                --out_vendor out/target/product/vsoc_x86_64/vendor                --out_odm out/target/product/vsoc_x86_64/odm     --out_product out/target/product/vsoc_x86_64/product             > out/target/product/vsoc_x86_64/host_init_verifier_output.txt"
host_init_verifier: out/target/product/vsoc_x86_64/vendor/etc/init/init.cutf_cvm.rc: 108: ignored duplicate definition of service 'bugreport'
host_init_verifier: Failed to parse init scripts with 1 error(s).

Change-Id: I0af60a1c19035d336a88a543089fd01400736dd5
---
 prebuilt/common/etc/init/init.lineage-system_ext.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/prebuilt/common/etc/init/init.lineage-system_ext.rc b/prebuilt/common/etc/init/init.lineage-system_ext.rc
index fdee7ea7..9a7fbed9 100644
--- a/prebuilt/common/etc/init/init.lineage-system_ext.rc
+++ b/prebuilt/common/etc/init/init.lineage-system_ext.rc
@@ -3,7 +3,7 @@ on init
     export TERMINFO /system_ext/etc/terminfo
 
 # bugreport is triggered by holding down volume down, volume up and power
-service bugreport /system/bin/dumpstate -d -p -z
+service lineage-bugreport /system/bin/dumpstate -d -p -z
     oneshot
     disabled
     keycodes 114 115 116
-- 
2.39.5

